README.md

```diff
@@ -218,6 +218,144 @@ OpenStack | N | N | Y | N | N
 oVirt | N | N | Y | N | N
 VMWare | N | N | Y | N | N
 
+# Short lived Credentials with AWS Secure Token Service
+### Overview
+We can also make Openshift use temporary credentials for different components with AWS Secure Token Service (STS). It enables an authentication flow allowing a component to assume an IAM Role resulting in short-lived credentials. It also automates requesting and refreshing of credentials using OpenID Connect Identity Provider. Openshift can sign ServiceAccount token trusted by AWS IAM which can be projected into a Pod and used for authentication. The following is a diagram showing how it works.
+
+![sts flow](docs/sts_flow.png)
```

Comment:
My fault for having GitHub in dark mode, but the black text on transparent backdrop in this is hard to see :slightly_smiling_face:

Reply:
![image](https://user-images.githubusercontent.com/4999586/105910348-cfede500-5ff6-11eb-9718-09efe047712b.png)
Here it is with a white background.

Reply:
@akhil-rane @joelddiaz ,  a digression, if we want to do this sts installation, the users also need to use an extra credentials or token that assuming role, because when we use the sts-preflight tool, it will create s3 bucket or IAM roles  on aws, or when we generate install-config and manifests, we still need to interact with the platform.
until the new sts token created for installation, then we can use it to finish our installation.  Is my understanding right?

Reply:
Yes. There is the initial AWS credentials that are used to create all this infrastructure have to be functional before any of these steps can be followed. So perhaps something near the beginning that says "you're going to need some powerful credentials to create all the infrastructure necessary for using STS".

Reply:
Included the new diagram. Also included comment saying sts-preflight needs creds with sufficient permissions